packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -175,6 +236,70 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     }
   }
 
+  function commitErrorLogging(
+    finishedWork: Fiber,
+    onUncaughtError: (error: Error) => void,
+  ) {
+    switch (finishedWork.tag) {
+      case ClassComponent:
+        {
+          const ctor = finishedWork.type;
+          const instance = finishedWork.stateNode;
+          const updateQueue = finishedWork.updateQueue;
+          invariant(
+            updateQueue !== null && updateQueue.capturedValues !== null,
+            'An error logging effect should not have been scheduled if no errors ' +
+              'were captured. This error is likely caused by a bug in React. ' +
+              'Please file an issue.',
+          );
+          const capturedErrors = updateQueue.capturedValues;
+          updateQueue.capturedValues = null;
+
+          if (typeof ctor.getDerivedStateFromCatch !== 'function') {
+            // To preserve the preexisting retry behavior of error boundaries,
+            // we keep track of which ones already failed during this batch.
+            // This gets reset before we yield back to the browser.
+            // TODO: Warn in strict mode if getDerivedStateFromCatch is
+            // not defined.
+            markLegacyErrorBoundaryAsFailed(instance);
+          }
+
+          instance.props = finishedWork.memoizedProps;
+          instance.state = finishedWork.memoizedState;
+          for (let i = 0; i < capturedErrors.length; i++) {
+            const errorInfo = capturedErrors[i];
+            const error = errorInfo.value;
+            logError(finishedWork, errorInfo);
+            instance.componentDidCatch(error);
```

Comment:
Did we intentionally "lose" the `errorInfo` argument here? Seems like a major change.